res/_headers

```diff
@@ -0,0 +1,31 @@
+# Headers for the entire site:
+/*
```

Comment:
At first read, I actually read this as a comment. From the discussion in #perf I realized what this meant :) But maybe the comment above should have something like : "/* represents any URL"

Reply:
Done.